tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -23,35 +23,55 @@ internal static class Debugger
             /// Default value is 1.
             /// </summary>
             /// <seealso cref="DebuggerSettings.MaximumDepthOfMembersToCopy"/>
-            public const string MaxDepthToSerialize = "DD_DEBUGGER_MAX_DEPTH_TO_SERIALIZE";
+            public const string MaxDepthToSerialize = "DD_DYNAMIC_INSTRUMENTATION_MAX_DEPTH_TO_SERIALIZE";
 
             /// <summary>
             /// Configuration key for the maximum duration (in milliseconds) to run serialization for probe snapshots.
             /// Default value is 150 ms.
             /// </summary>
             /// <seealso cref="DebuggerSettings.MaxSerializationTimeInMilliseconds"/>
-            public const string MaxTimeToSerialize = "DD_DEBUGGER_MAX_TIME_TO_SERIALIZE";
+            public const string MaxTimeToSerialize = "DD_DYNAMIC_INSTRUMENTATION_MAX_TIME_TO_SERIALIZE";
 
             /// <summary>
             /// Configuration key for the maximum upload batch size.
             /// Default value is 100.
             /// </summary>
             /// <seealso cref="DebuggerSettings.UploadBatchSize"/>
-            public const string UploadBatchSize = "DD_DEBUGGER_UPLOAD_BATCH_SIZE";
+            public const string UploadBatchSize = "DD_DYNAMIC_INSTRUMENTATION_UPLOAD_BATCH_SIZE";
```

Comment:
Is this change in configuration key going to break users? Also, I like how the new symbol database batch size specifies that it's in bytes. What is the unit of this "size" and, if it makes sense, can the units be included in the configuration name?

Reply:
I think that is less make sense here because there is no size unit here but simply a counter of how much do we want to upload on each batch.
@OmerRaviv what about the first question in @zacharycmontoya comment, is it safe to do that change?

Reply:
We agreed it was safe to change